### PR TITLE
Don't show empty tooltips

### LINKED ISSUE #3003: Tooltip shows weird empty container when no text available
screencast 2018-07-11 15-04-07

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/60734540-tooltip-shows-weird-empty-container-when-no-text-available?utm_campaign=plugin&utm_content=tracker%2F4707269&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F4707269&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by HoutarouOreki**
Resolves #3003.

---

### CODE REVIEW

File: osu.Game/Graphics/Cursor/OsuTooltipContainer.cs [hunk 1]
```diff
@@ -70,6 +71,7 @@ public OsuTooltip()
                         TextSize = text_size,
                         Padding = new MarginPadding(5),
                         Font = @"Exo2.0-Regular",
+                        Text = " ",
```
Comment (peppy) [inline osu.Game/Graphics/Cursor/OsuTooltipContainer.cs]: what is this?

Comment (HoutarouOreki) [reply osu.Game/Graphics/Cursor/OsuTooltipContainer.cs]: Text needs to be changed so that the first tooltip wont show if text is empty. Basically

Comment (peppy) [reply osu.Game/Graphics/Cursor/OsuTooltipContainer.cs]: this shouldn't be necessary / can't exist

**Comment by ekrctb** [comment]
This PR doesn't fix the issue.

**Pull request closed without merging**